libtelemetry/src/main/java/com/mapbox/android/telemetry/MapboxTelemetry.java

```diff
@@ -36,6 +36,7 @@
   private static final String NAVIGATION_USER_AGENT = "mapbox-navigation-android/";
   private static final String NAVIGATION_UI_USER_AGENT = "mapbox-navigation-ui-android/";
   private static final String MAPS_USER_AGENT = "mapbox-maps-android/";
+  private static final String VISION_USER_AGENT = "MapboxVision/";
```

Comment:
should we be consistent with user agent format? why not `mapbox-vision-android`?

Reply:
This was the requested change from stakeholder (@BenDenen) and matches the iOS implementation, per slack discussion with @l-r. Additionally, we don't actually have a set format, with `MapboxEventsAndroid`, `MapboxTelemetryAndroid`, and `MapboxEventsUnityAndroid` all being valid user agents. 

The tweak I would think to make, if it is ok with everyone involved, is to either add the hyphen system (suggested above) or add `Android` to end of `MapboxVision` (MapboxVisionAndroid) to help differentiate between iOS and android implementations. If that seems unnecessary, since we send back platform with events, then I am fine with how it is now.

Reply:
> Additionally, we don't actually have a set format, with MapboxEventsAndroid, MapboxTelemetryAndroid, and MapboxEventsUnityAndroid all being valid user agents.

If that's true, what's the point in maintaining `VALID_USER_AGENTS`? And would it make more sense to pass through network error if `userAgent` field is invalid (provided that backend will return error code for invalid `userAgent`)?